include/inviwo/core/properties/minmaxproperty.h

```diff
@@ -272,29 +272,27 @@ MinMaxProperty<T>& MinMaxProperty<T>::setMinSeparation(const T& newMinSeparation
 }
 
 template <typename T>
-MinMaxProperty<T>& MinMaxProperty<T>::setRange(const range_type& value) {
-    const auto newRange = clamp({glm::min(value.x, value.y), glm::max(value.x, value.y)});
+MinMaxProperty<T>& MinMaxProperty<T>::setRange(const range_type& newRange) {
+    if (range_.update({glm::min(newRange.x, newRange.y), glm::max(newRange.x, newRange.y)})) {
+        value_.update(clamp(value_));
 
-    if (value_.update(newRange)) {
         this->propertyModified();
         onRangeChangeCallback_.invokeAll();
     }
     return *this;
 }
 
 template <typename T>
-void MinMaxProperty<T>::set(const range_type& value, const range_type& range, const T& increment,
-                            const T& minSep) {
-
-    const auto newRange = clamp({glm::min(range.x, range.y), glm::max(range.x, range.y)});
+void MinMaxProperty<T>::set(const range_type& newValue, const range_type& newRange,
+                            const T& newIncrement, const T& newMinSep) {
 
     bool modified = false;
-    modified |= range_.update(newRange);
+    modified |= range_.update({glm::min(newRange.x, newRange.y), glm::max(newRange.x, newRange.y)});
```

Comment:
Again, is this not an input error?

Reply:
this is how we have used to do it, if you want to throw go ahead and add that...